test/core/fork/NFTDescriptor.t.sol

```diff
@@ -28,15 +28,15 @@ contract NFTDescriptor_Fork_Test is Fork_Test {
 
     /// @dev Loads the Lockup V2.0 contracts pre-deployed on Mainnet.
     modifier loadDeployments_V2_0() {
-        lockupDynamic = ISablierV2LockupDynamic(0x39EFdC3dbB57B2388CcC4bb40aC4CB1226Bc9E44);
-        lockupLinear = ISablierV2LockupLinear(0xB10daee1FCF62243aE27776D7a92D39dC8740f95);
+        lockupDynamic = ISablierLockupDynamic(0x39EFdC3dbB57B2388CcC4bb40aC4CB1226Bc9E44);
+        lockupLinear = ISablierLockupLinear(0xB10daee1FCF62243aE27776D7a92D39dC8740f95);
         _;
     }
 
     /// @dev Loads the Lockup V2.1 contracts pre-deployed on Mainnet.
     modifier loadDeployments_V2_1() {
```

Comment:
let's change the `V2.1` to package version, i.e. `v1.1.2`

Reply:
Sounds good.